src/renderer/components/+workloads-pods/details/volumes/variants/__snapshots__/projected.test.tsx.snap

```diff
@@ -0,0 +1,229 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Projected /> renders 1`] = `
```

Comment:
I know that these are not exhaustive tests. We can add some more in the future I think.